### PR TITLE
feat(query): Added Path Parameter With No Corresponding Template Path (v2) query to Swagger 2.0

### PR DESCRIPTION
**Pull request by joaoReigota1**
**Proposed Changes**
- Added Path Parameter With No Corresponding Template Path (v2) query to Swagger 2.0

I submit this contribution under the Apache-2.0 license.

### CODE REVIEW

**Comment by joaoReigota1** [comment]
LGTM

**Review by felipe-avelar — CHANGES_REQUESTED** [review]
Needs to add "(v3)" to `queryName`

File: assets/queries/openAPI/general/path_parameter_with_no_corresponding_template_path/test/positive_expected_result.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
+[
+  {
+    "queryName": "Path Parameter With No Corresponding Template Path",
```
Comment (rogeriopeixotocx) [inline assets/queries/openAPI/general/path_parameter_with_no_corresponding_template_path/test/positive_expected_result.json:3]: v3 missing

**Review by rogeriopeixotocx — DISMISSED** [review]

**Review by felipe-avelar — APPROVED** [review]
LGTM

**Pull request merged**